.circleci/config.yml

```diff
@@ -439,7 +439,11 @@ job_configuration:
     image: marcotc/docker-library:ddtrace_rb_3.0.0
     jit: true
   # JRuby
-  - &config-jruby-9_2
+  - &config-jruby-9_2_0 # Oldest supported version
+      <<: *filters_all_branches_and_tags
+      ruby_version: 'jruby-9.2.0'
+      image: marcotc/docker-library:ddtrace_rb_jruby_9_2_0
```

Comment:
Perhaps call this `jruby-9_2_0_0` everywhere, just to match how JRuby sets its versions?

Reply:
I like the naming suggestion, performed both.